Extension/src/Debugger/attachToProcess.ts

```diff
@@ -187,13 +187,18 @@ export class RemoteAttachPicker {
         const args: string[] = [`-ex "target extended-remote ${miDebuggerServerAddress}"`, '-ex "info os processes"', '-batch'];
         let processListOutput: util.ProcessReturnType = await util.spawnChildProcess(miDebuggerPath, args);
         // The device may not be responsive for a while during the restart after image deploy. Retry 5 times.
-        for (let i: number = 0; i < 5 && !processListOutput.succeeded; i++) {
+        for (let i: number = 0; i < 5 && !processListOutput.succeeded && processListOutput.outputError.length === 0; i++) {
             processListOutput = await util.spawnChildProcess(miDebuggerPath, args);
         }
 
         if (!processListOutput.succeeded) {
             throw new Error(localize('failed.to.make.gdb.connection', 'Failed to make GDB connection: "{0}".', processListOutput.output));
         }
+
+        if (processListOutput.outputError.length !== 0) {
+            throw new Error(localize('failed.to.make.gdb.connection', 'Failed to make GDB connection: "{0}".', processListOutput.outputError));
```

Comment:
@bobbrow @Colengms Is 2 of the same loc string okay?

Reply:
Oh, I see 2 of the same loc label in other files, so it seems like it's okay.

Reply:
It is ok if the string is identical in both uses.  Otherwise, you may not get the string you expect.